control_panel_api/services.py

```diff
@@ -175,6 +175,12 @@ def create_bucket(bucket_name):
         target_prefix=f"{bucket_name}/")
     aws.put_bucket_encryption(bucket_name)
 
+    if is_data_warehouse:
```

Comment:
One thing I wonder is if we should tag the non-data-warehouse buckets (with `{'bucket-type': 'app'}`) or not. I suppose the "problem" with that would be that then some app buckets have this tag but some they don't.

It probably doesn't matter at this stage as we don't use these tags anyway.